Initialize-ApimDeveloperPortal.ps1

```diff
@@ -0,0 +1,56 @@
+<#
+.SYNOPSIS
+Provisions and publishes the environments' APIM Developer portals with default styling.
+
+.DESCRIPTION
+Provisions and publishes the environments' APIM Developer portals with default styling.
+It is designed to be ran by an Azure CLI task with Script Type Powershell Core on a deployment agent.
+If release with source alias for https://github.com/Azure/api-management-developer-portal/ is api-management-developer-portal,
+working directory for Azure CLI task is set to $(System.DefaultWorkingDirectory)/api-management-developer-portal/scripts.v3
+
+.PARAMETER EnvironmentNames
+A string array of environments in a subscription. For example:
+"AT,TEST"
+
+#>
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $true)]
+    [ValidateSet("DTA", "AT", "TEST", "TEST2", "DEMO", "PP", "PRD", "MO")]
+    [string[]]$EnvironmentNames
+)
+
+$AzureSubscriptionId = (az account show | ConvertFrom-Json).id
+
+Copy-Item -Path ./media -Destination ./../dist/snapshot/media
+Copy-Item -Path ./media/* -Destination ./../dist/snapshot/media
+Copy-Item -Path ./data.json -Destination ./../dist/snapshot
+Get-ChildItem -Path ./../dist/snapshot/media
+Get-ChildItem -Path ./../dist/snapshot
+
+foreach ($EnvironmentName in $EnvironmentNames) {
+
+    $ApimResourceGroupName = "das-$($EnvironmentName)-apim-rg"
```

Comment:
Could the EnvironmentName be converted to lower? Just to remove any errors when getting the azure resources/doing the POST to the developer portal address as well

Reply:
Done